modules/ding_debt/plugins/content_types/debts/debts.inc

```diff
@@ -84,6 +89,11 @@ function ding_debt_debts_form($form, &$form_state, $debts, $has_invoiced_fees, $
         '#value' => t('Pay selected debts'),
         '#suffix' => '</div>',
         '#submit' => array('ding_debt_debts_form_submit_pay_selected'),
+        '#states' => array(
+          'disabled' => array(
+            ':input[name="accept_terms"]' => array('checked' => FALSE),
```

Comment:
:+1: for using `#states`.

If I recall correctly states is clientside only so in theory a user could enable the button clientside and not accept the terms of service. I do not think it is hugely important here but you could consider adding `#required`.

Reply:
@kasperg good catch. In that sense the #states => disabled is only for visual purposes. I 
updated the PR with #required on checkbox.

It will show the red asteriks by the checkbox, but i don't think that matters. Also the form error message is very generic. I don't think it's worth providing a custom error message, since we don't expect users to do this.

![4161-accept-required](https://user-images.githubusercontent.com/5011234/65317480-34e03200-db9c-11e9-9994-c813fbab4c87.png)